build.sh

```diff
@@ -434,7 +433,8 @@ if [ ${BUILD_BE} -eq 1 ]; then
     if [ "${BUILD_TYPE}" == "ASAN" ]; then
         cp -r -p ${STARROCKS_HOME}/be/output/conf/asan_suppressions.conf ${STARROCKS_OUTPUT}/be/conf/
     fi
-    cp -r -p ${STARROCKS_HOME}/be/output/lib/* ${STARROCKS_OUTPUT}/be/lib/
+    cp -r -p ${STARROCKS_HOME}/be/output/lib/starrocks_be ${STARROCKS_OUTPUT}/be/lib/
+    cp -r -p ${STARROCKS_HOME}/be/output/lib/libmockjvm.so ${STARROCKS_OUTPUT}/be/lib/libjvm.so
```

Comment:
better name 'stubjvm.so' ?  
'mockjvm.so' is also fine.

Reply:
because ./starrocks_be has link with libjvm.so we also need make mock.so require the same name

Reply:
```
>ldd ./starrocks_be 
        linux-vdso.so.1 (0x00007ffd4edbc000)
        libjvm.so => not found
        libbfd-2.38-system.so => /lib/x86_64-linux-gnu/libbfd-2.38-system.so (0x00007fcf0fd2c000)
        libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007fcf0fc45000)
        libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007fcf0fa1d000)
        /lib64/ld-linux-x86-64.so.2 (0x00007fcf0feb4000)
```